General/Majority Element.py

```
@@ -0,0 +1,8 @@
+class Solution:
+    def majorityElement(self, nums: List[int]) -> int:
```

Ruff (0.11.9)

Undefined name `List`

(F821)